u19_pipeline/imaging_pipeline.py

```diff
@@ -70,15 +70,15 @@ class ScanInfo(dj.Imported):
 
 
 @schema
-class FieldOfView(dj.Imported):
+class TiffSplit(dj.Imported):
     definition = """
     # meta-info about specific FOV within mesoscope imaging session
     # `make` function is declared in the `U19-pipeline-matlab` repository
     -> ImagingPipelineSession
-    fov                  : tinyint                      # number of the field of view in this scan
+    tiff_split           : tinyint                      # number of the tiff split in this scan
     ---
-    fov_directory        : varchar(255)                 # the absolute directory created for this fov
-    fov_name=null        : varchar(32)                  # name of the field of view
+    tiff_split_directory : varchar(255)                 # the absolute directory created for this tiff_split
+    tiff_split_name=null : varchar(32)                  # name of the tiff_split
     fov_depth            : float                        # depth of the field of view  should be a number or a vector?
```

Comment:
I believe this would be a number, not a vector. Possibly could be simplified as `int` instead of `float`, but not certain

Reply:
Sorry for ambiguity - line 81:
```
fov_depth            : float                        # depth of the field of view  should be a number or a vector?
```

Reply:
Apparently this fov_depth is some measurement (I guess in mm) 
From our u19_imaging.FieldOfView table:
<img width="117" alt="Screen Shot 2022-06-09 at 10 14 15 AM" src="https://user-images.githubusercontent.com/56653225/172868670-26b85e6f-ddf9-4dc1-bf3d-7dfbe322dbc7.png">

Reply:
Thanks @Alvalunasan.  Then we can keep it as a float.